master_xml.py

```diff
@@ -124,6 +124,9 @@ def evaluateInfoData(cards):
                             if abilityValue != None:
                                 cards[cardId]['info'][region] = cards[cardId]['info'][region].replace("{" + key + "}", abilityValue)
 
+            cards[cardId]['infoFormatted'][region] = cards[cardId]['info'][region]
```

Comment:
consider naming the field after the format, e.g. `infoXML`  perhaps other formats will exist